Src/WitsmlExplorer.Frontend/components/ContentViews/table/ColumnDef.tsx

```diff
@@ -17,6 +19,12 @@ declare module "@tanstack/react-table" {
 export const useColumnDef = (viewId: string, columns: ContentTableColumn[], insetColumns: ContentTableColumn[], checkableRows: boolean, stickyLeftColumns: number) => {
   const isCompactMode = useTheme().props.MuiCheckbox?.size === "small";
 
+  const {
+    operationState: { decimals },
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    dispatchOperation
+  } = useContext(OperationContext);
```

Comment:
No need to disable the eslint warning, we're not using dispatchOperation here anyways, so it can just be removed.

`const { operationState: { decimals } } = useContext(OperationContext);`